templates/config.json.j2

```diff
@@ -1,25 +1,27 @@
 {
-    "masters": [{% for host in groups['mesosmaster'] %}"{{hostvars[host]['ansible_default_ipv4']['address']}}:5050"{% if not loop.last %},{% endif %}{% endfor %}],
```

Comment:
this add dependency on hosts to have [mesosmaster] I have changed it to {{mesos_dns_zookeerpers}} variable and added default to listen for localhost